src/client.coffee

```diff
@@ -221,7 +221,7 @@ class Client extends EventEmitter
                 @emit 'new_user', message
             else
                 # Check for `pong` response
-                if message.data?.text? and message.data.text == "pong"
+                if message.data.text? and message.data.text == "pong"
```

Comment:
@Vhex I'll happily merge the httpPort stuff, also in github-matteruser but please revert the typo changeset, or split it out in a separate PR if you like to debate it :)